templates/engage/edge-month.html

```diff
@@ -0,0 +1,115 @@
+{% extends_with_args "engage/base_engage.html" with title="A guide to edge computing and the tools you need" meta_image="https://assets.ubuntu.com/v1/cdbfad95-Social+banner.jpg" meta_description="Join Canonical Product Managers for a series of webinars on edge computing to learn what is driving businesses to process their data at the edge and how you can add an edge computing layer to your business infrastructure. " %}
+
+{% block meta_copydoc %}https://docs.google.com/document/d/1D_CuxaO_II0Jh9jRO9mYgRnO1J99s3DY-Iq78fPpvTc{% endblock meta_copydoc %}
+
+{% block content %}
+
+<section class="p-strip p-strip--dark p-takeover--edge">
+  <div class="row u-equal-height">
+    <div class="col-8 u-vertically-center">
+      <div>
+      <h1>
+        Canonical presents:<br>Edge month
+      </h1>
+      <h4>
+        Join us for a month of webinars on edge computing.  Learn how edge computing provides enterprises with real-time data processing, cost-effective security and scalability.
+      </h4>
+      </div>
+    </div>
+  </div>
+</section>
+
+<section class="p-strip is-shallow is-bordered">
+    <div class="row">
+      <div class="col-6">
+        <div class="p-heading-icon--muted">
+          <div class="p-heading-icon__header">
+            <img class="p-heading-icon__img p-heading-icon__img--small" src="{{ ASSET_SERVER_URL }}26797d10-newsfeed-stripvideo-icon.svg" width="32" alt="" />
+            <h4 class="p-heading-icon__title">Webinar</h4>
```

Comment:
Similarly, I don't think these elements need to be a `h4`, since it's closely followed by a `h3` with more important info - it could be a `p` tag.